src/main/kotlin/lotto/domain/ProfitRate.kt

```diff
@@ -0,0 +1,31 @@
+package lotto.domain
+
+import java.math.RoundingMode
+import java.text.DecimalFormat
+
+class ProfitRate(
+    private val matchResult: Map<Ranking, Int>,
+    private val inputPrice: Int
+) {
+
+    fun calculate(): Double {
+        val totalWinningMoney = getTotalWinningMoney()
+        return calculate(totalWinningMoney)
+    }
+
+    private fun getTotalWinningMoney(): Int {
+        return matchResult.entries
+            .sumOf { it.key.getTotalWinningMoney(it.value) }
+    }
+
+    private fun calculate(totalWinningMoney: Int): Double {
+        val decimalFormat = DecimalFormat(PROFIT_RATE_PATTERN)
+        decimalFormat.roundingMode = RoundingMode.DOWN
+        val profitRate = totalWinningMoney / inputPrice.toFloat()
```

Comment:
어짜피 `Double`로 반환하려는건데 여기서는 float로 변경하신 이유가 있으실까요? 🤔 
```suggestion
        val profitRate = totalWinningMoney / inputPrice.toDouble()
```

Reply:
오타입니다 ㅠㅠ 자세히 봐주셔서 감사합니다 😀